src/parser/events-assets/parser.unit.test.ts

```diff
@@ -149,5 +149,64 @@ describe('Query parser', () => {
 
       expect(parsedWithSpaces).toEqual(parsedWithoutSpaces);
     });
+    it('should parse escape characters in single quoted string', () => {
+      const querySBS = String.raw`assets{metadata={key1!~'\value2 \d \. - \w+', key2!~"\value2 \d \. - \w+"}}`;
```

Comment:
probably neat-picking, but maybe we could 
destruct string to get `raw` function? so then we don't need to be that verbose every time

```
const { raw: r } = String;
```
then it will look like in python)))

Reply:
haha, let's try)